README.md

```diff
@@ -25,6 +25,7 @@ To run the front-end, run the command `yarn dev` from the `UPRMeals/frontend` di
 ## Backend
 
 To run the back-end, run the command `yarn start:dev` from the `UPRMeals/backend` directory.
+Make sure you have the database container running and have the latest migrations applied, to do this run `yarn prisma migrate dev` inside the backend directory.
```

Comment:
Can you specify how to run the database container?

```suggestion
To run the database container, ... .
Always make sure you have latest migrations applied, to apply migrations run `yarn prisma migrate dev` inside the backend directory.
```

Reply:
thats already on the readme in the Datase section that you created, lo muevo aqui?